wallet/wallet.go

```diff
@@ -3315,6 +3314,27 @@ func (w *Wallet) publishTransaction(tx *wire.MsgTx) (*chainhash.Hash, error) {
 	}
 }
 
+// HasSpendingTx determines whether the store contains a spending transaction
+// for the given outpoint. This transaction can be either confirmed or
+// unconfirmed.
+//
+// NOTE: It's possible for there not to be a spending transaction for this
+// outpoint, therefore a nil check must be used to guarantee safety to the
+// caller.
+func (w *Wallet) HasSpendingTx(op wire.OutPoint) (*wire.MsgTx, error) {
```

Comment:
Now that the boolean is gone, we should probably change the name of the func too. `GetWalletSpendingTx` or something? (maybe reference wallet too to distinguish from getting any spending tx)

Reply:
`SpendingTx` should be more in line with the standard Go names I reckon.